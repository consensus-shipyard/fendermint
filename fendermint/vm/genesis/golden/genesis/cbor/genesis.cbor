a56c6e6574776f726b5f6e616d65670f2f56efbfbd3e6f6e6574776f726b5f76657273696f6e1268626173655f666565587d00ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffba585791485be44ffce5af53ce67fecf491f8457a787249212dbc0220a5b35386a76616c696461746f72738aa26a7075626c69635f6b65795841046184b1e6b3ba2b5a3993924bcfe69117adbc9f814e50dfbe4e6cd7442b831dc05c6f4e0ef2576b97dff32cb1b8c6eab5caa230d1afa98ed29ea39a9174e9b91f65706f7765721b2c4ed59b58645aaaa26a7075626c69635f6b657958410474d359d331a28592010ad1c710b711fa7e03b91521e666dc5646bf7236cfe5041453995dea1d56f27d50cb42c318077fbb9442fbec0bd2c38476d1342d1a980565706f7765721b12d344507fdc1aada26a7075626c69635f6b6579584104542acf24055ecbd0fc1665c1de767bcc88bd5abd83bbd8c08fbc87fba382a4060e7c1a0014450708a13b0d039739495782173d8da4539950224a25b1b91f5cfb65706f7765721bc69e0132e5d68a12a26a7075626c69635f6b6579584104a42e4ef7b108ba00a9721a438e853501b5eb28110baf76bcc10b799e8bae2df463b04a02063a0ff26925250bc27e427264064d0d19fbdd62d96b98fbd63af3a265706f7765721bc4f475d3d183fbf2a26a7075626c69635f6b6579584104c40211345cdf16660d21e380f535b7126089c283848ec67925ba5a55ac494a56ee9dd8517edb02f6c71e90f2d12676ad45bfac137dc7f2da9f86a6317ccb20ac65706f7765721b0e606d239cafc551a26a7075626c69635f6b6579584104da9bdf583e2e8906229a1708978f5c35504deaaa62ca77b63f986b45525b9b2764178ff54441fd8733fac65e37989ed7e962d85ce6824842c702b17c0fa0027e65706f7765721b08e0d6fa82437ccea26a7075626c69635f6b6579584104f94fad33f2040693d8aac32a2f9b5586ac355f3f373cb5af7a93fe15e001067c92ad63ab3f8eda6770b97d50618518c7bf50a521c62f5ce94f90fc19ccd71c8b65706f77657200a26a7075626c69635f6b65795841040a64bea40fe4a0ba4949263f0d65594372dd73c88d5fba4c9ad911698179ed1cd4cb9f7637eddb0dd4082b582d30543ee8f737081ca77581390ac72f8fb9817065706f7765721b0cbc1519d0fdc605a26a7075626c69635f6b65795841043dc89b4d180743a7477c9792c40383e8fd8114e92757a20b265f4a96c8e2551f909809afaad83702e07ca98baf512524e479e502e8aa9c41ddfbf6ef482270a765706f7765721b47e52446ff607d0ea26a7075626c69635f6b6579584104b0737e644c27fd8e1acb15d86154ac73df61ab336a4c0e4610f34a0c9b395d2d44ef59241f21783fc689267cffa751d6ad13acd26d6b6b83f3f02b120fc1518865706f7765721b3f3a401081848a7f686163636f756e747381a2646d657461a1684d756c7469736967a4677369676e657273834b00d69dd6c1a69afceff7014b00bc9e849baea9cf8ddc0155010501dfeda38c880ef884997c5927925d002c75dc697468726573686f6c64017076657374696e675f6475726174696f6e1bf5c2105075b9a8386d76657374696e675f73746172741bdbaf5d54c80e4f496762616c616e6365587d00ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff01fe61432e4db587d069da19c22721726eff68896b8ec8eb4b68c5b559f471dbffffffffffffffff4d35c11d22fe62a8869b6c52a49fe6a879e5f54918c76abcffffffffffffffff