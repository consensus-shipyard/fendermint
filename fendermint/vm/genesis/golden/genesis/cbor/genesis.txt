Genesis { network_name: "\u{f}/Vï¿½>", network_version: NetworkVersion(18), base_fee: TokenAmount(41855804968213567224547853478906320725054875457247406540771499545716837934567817284890561672488119458109166910841919797858872862722356017328064756151166307827869405370407152286801072676024887272960758524035337792904616958044270623287606981302907767513269212945003262782944539768968.35359063973371628), validators: [Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [58924480, 53858570, 33285350, 33896771, 61717663, 66692165, 20522172, 11364582, 31896506, 1597740], magnitude: 1, normalized: true }, y: Field { n: [15317279, 15115357, 15542762, 54967974, 30057008, 36813485, 53660443, 28204927, 34533975, 1514451], magnitude: 1, normalized: true }, infinity: false })), power: Power(3192724049031879338) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [47179012, 28302477, 40748388, 5539737, 41812921, 2999422, 11345009, 35014660, 30618018, 1914070], magnitude: 1, normalized: true }, y: Field { n: [18520069, 28593419, 19675207, 66039855, 62624834, 12976607, 17609772, 22792693, 22931997, 333030], magnitude: 1, normalized: true }, infinity: false })), power: Power(1356503025301854893) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [58893318, 52559592, 25954555, 49680111, 8961370, 60661491, 23485469, 53429232, 52692318, 1378995], magnitude: 1, normalized: true }, y: Field { n: [18832635, 42560622, 26542628, 37130574, 58857277, 30298709, 61919289, 1843844, 33559621, 237318], magnitude: 1, normalized: true }, infinity: false })), power: Power(14311877983946050066) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [61746676, 48129954, 57396240, 4468413, 28699432, 60902720, 52536376, 48759461, 49787144, 2689939], magnitude: 1, normalized: true }, y: Field { n: [37417890, 48643829, 30813590, 3434479, 40109645, 10457244, 38949052, 4180388, 33687098, 1633298], magnitude: 1, normalized: true }, infinity: false })), power: Power(14192097878364912626) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [4803158, 43423083, 6787675, 34476603, 39881154, 21851588, 35534863, 5871668, 20208863, 3211396], magnitude: 1, normalized: true }, y: Field { n: [13312172, 27888735, 53324280, 5109535, 21348268, 4824491, 32050989, 776988, 5340891, 3909494], magnitude: 1, normalized: true }, infinity: false })), power: Power(1035947914015262033) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [39557927, 35311956, 58418169, 44665641, 22040042, 31708941, 27357321, 35920010, 56114734, 3581687], magnitude: 1, normalized: true }, y: Field { n: [60818046, 11296515, 8662128, 24353289, 65626840, 31860661, 61629923, 66460879, 66405441, 1639907], magnitude: 1, normalized: true }, infinity: false })), power: Power(639747518502239438) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [67196, 16745848, 56293289, 16571634, 44840287, 65459553, 44839586, 1724258, 20181508, 4084715], magnitude: 1, normalized: true }, y: Field { n: [14097547, 4130419, 30315769, 8853693, 62869669, 6374961, 60282118, 57253314, 61554574, 2403160], magnitude: 1, normalized: true }, infinity: false })), power: Power(0) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [24767772, 38034016, 61131181, 52573566, 57859443, 56186448, 9593840, 42133797, 44306404, 170287], magnitude: 1, normalized: true }, y: Field { n: [62488944, 45206499, 56103824, 2126493, 48822071, 55317775, 8566146, 57423696, 58079213, 3486439], magnitude: 1, normalized: true }, infinity: false })), power: Power(917631624701396485) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [14832927, 64136626, 35697253, 61119838, 16613652, 16834810, 63535404, 17734941, 55384071, 1012262], magnitude: 1, normalized: true }, y: Field { n: [2257063, 50183122, 29629919, 762538, 14973413, 64244041, 63609018, 14420865, 28289752, 2369026], magnitude: 1, normalized: true }, infinity: false })), power: Power(5180586833710447886) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [54091053, 13796134, 14967055, 13478192, 64971179, 5581596, 11623814, 66467947, 40127527, 2890975], magnitude: 1, normalized: true }, y: Field { n: [63000968, 705667, 45629247, 55162285, 44897196, 65655925, 9594831, 31522586, 19144481, 1129430], magnitude: 1, normalized: true }, infinity: false })), power: Power(4556024412675279487) }], accounts: [Actor { meta: Multisig(Multisig { signers: [ActorAddr(Address { payload: ID(17861259433656815318) }), ActorAddr(Address { payload: ID(15860337904566996796) }), ActorAddr(Address { payload: Secp256k1([5, 1, 223, 237, 163, 140, 136, 14, 248, 132, 153, 124, 89, 39, 146, 93, 0, 44, 117, 220]) })], threshold: 1, vesting_duration: 17708734622532741176, vesting_start: 15829973833945599817 }), balance: TokenAmount(41855804968213567224547853478906320725054875457247406540771499545716837934567817284890561672488119458109166910841919797858872617318109227516801281035464999484940162850910224733408517982449299255832557931560327131699776021284211056730173133631077042112880094222012962341995919605640.812416069877628927) }] }